test/e2e/main/custom_configuration_test.go

```diff
@@ -200,3 +207,32 @@ xmlns:server="urn:infinispan:server:13.0">
 		Data: map[string]string{"infinispan-config." + format: userCacheContainer},
 	}
 }
+
+func addLog4jToCustomConfigMap(customConfigMap *corev1.ConfigMap) *corev1.ConfigMap {
+	// Add custom Log4j config to existing ConfigMap
+	customConfigMap.Data["log4j.xml"] = `<?xml version="1.0" encoding="UTF-8"?>
+<Configuration
+        name="InfinispanServerConfig" monitorInterval="60" shutdownHook="disable">
+    <Appenders>
+        <!-- Colored output on the console -->
+        <Console name="STDOUT">
+            <PatternLayout pattern="{{ .Logging.Console.Pattern }}"/>
+        </Console>
+    </Appenders>
+
+    <Loggers>
+        <Root level="INFO">
+            <AppenderRef ref="STDOUT" level="{{ .Logging.Console.Level | UpperCase }}"/>
+        </Root>
+
+        {{ range $key, $value := .Logging.Categories }}
+        <Logger name="{{ $key }}" level="{{ $value | UpperCase }}"/>
+        {{ end }}
+    </Loggers>
+</Configuration>`
```

Comment:
This xml contains go template logic which won't be processed before the file is mounted to the server pod, resulting in an invalid logging configuration. You should remove all content in the `{{  }}` brackets and replace with the actual values to be used.

The xml below is valid and it configures the server so that all infinispan packages are logged at the trace level.

```xml

<?xml version="1.0" encoding="UTF-8"?>
<Configuration name="InfinispanServerConfig" monitorInterval="60" shutdownHook="disable">
    <Appenders>
        <!-- Colored output on the console -->
        <Console name="STDOUT">
            <PatternLayout pattern="%d{HH:mm:ss,SSS} %-5p (%t) [%c] %m%throwable%n"/>
        </Console>
    </Appenders>

    <Loggers>
        <Root level="INFO">
            <AppenderRef ref="STDOUT" level="TRACE"/>
        </Root>
        <Logger name="org.infinispan" level="TRACE"/>
    </Loggers>
</Configuration>
```

Reply:
cool, updated the config with the suggested sample, thanks!